### ISSUE TITLE
Remove browser focus highlight

### ISSUE THREAD

**Issue by billreed63** [maqetta/maqetta#1109]
The browsers put a highlight around the focused item, dojo widget have their own focus highlight. We should add the 
`*:focus { outline: none; }` to devinci.css to remove the browser focus

**Comment by JonFerraiolo** [comment]
@peller

I'm not sure about this one. Does Dojo/Dijit recommend setting outline:none for *:focus? What happens with HTML widgets that we have in our UI? (eg., prop palette uses raw HTML widgets)

**Comment by peller** [comment]
as I recall, the focus ring, whether native or manufactured, is an important consideration for a11y, so I suspect you'd never see it disabled in applications.   Bill, are you saying there are two focus rings visible?

**Comment by billreed63** [comment]
No, when a dojo widget is selected typically the color if the border around the widget is made a contrasting color... So of the times the browser focus is place around an child dom of the widget. When the user selects a tab for example the focus is placed around the name of the file, just looks a little goofy..

**Comment by JonFerraiolo** [comment]
yeah, it might look a little goofy, but I think it's a goofiness that we have to live with. 

After seeing Adam's comment, I thought some more about this and the browser focus highlights are essential not only for accessibility but also for normal user usage. Users just plain expect that they can hit tab and cycle through focusable items (UI controls and hyperlinks, mostly), and they expect that they will get visual feedback that highlights the thing that currently has the focus. In the Maqetta app, focusable items include mostly regular old HTML controls, but we do use some Dojo controls such as TabContainer. Going forward, I expect we will use fewer Dojo Controls and more HTML controls because CSS3 is becoming so rich. 

If we killed the focus highlighting for Dojo, we'd kill it also for HTML controls, and users would have no visual feedback for all of our HTML controls on where the focus is now. If we found a way to kill the focus highlighting just for Dojo controls, and leave it for HTML controls, then users would face inconsistency in how focus is represented depending on whether the current UI control is HTML or Dojo.

To me, the right thing is that Dojo's visual styling needs to take into account the focus highlighting used by the browsers to minimize any goofiness that results from browser highlighting happening simultaneously with Dojo highlighting.

**Comment by JonFerraiolo** [comment]
Pushing to M5. We might end up just closing this and accepting the way things are.

**Comment by JonFerraiolo** [comment]
Pushing to M6

**Comment by JonFerraiolo** [comment]
-> M7

**Comment by JonFerraiolo** [comment]
-> M8
